### PR TITLE
Feature/jcs/refactor

### LINKED ISSUE #28: Card Component - Add card component
- [ ] Card Component - added to render dynamic cities to home page
- [ ] Page should display the same as before 
- [ ] Move all styling to match previous

### LINKED ISSUE #29: AirQuality API and Proxy
- Connect AirQuality Api
- run proxy to prevent cors errors
- evaluate data from request 
- plan for function to clean data 
- plan for error handling functionality

### LINKED ISSUE #30: Remove View page route in APP
- REMOVE and add routing to app
- Move any styling attached to view component

### PR DESCRIPTION
**Pull request by JoshSevy**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Added airquality api added proxy to handle api card component and removed view component
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
Api is fetching data appropriately not stored yet until we know where the data is needed most
## Has This Been Tested?
- [ ] No
- [x] Yes -Browser no test files written yet
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
npm start
# Message for reviewer:
Site still operates and looks same as before just added that api and moved components around.
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs -in API to show data
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Weather Or Not</title>
+    <title>Goin SomewhAIR?</title>
```
Comment (ErinUntermeyer) [inline public/index.html:26]: <3

Comment (leighlars) [inline public/index.html:26]: Will experiment with differing names

File: src/App/App.scss [hunk 2]
```diff
@@ -7,3 +7,11 @@
   background: linear-gradient(0deg, rgba(51,138,242,0.9808298319327731) 13%, rgba(23,79,174,1) 41%, rgba(1,53,120,1) 59%);
   height: 100%;
 }
+
+main {
```
Comment (leighlars) [inline src/App/App.scss:11]: right on

File: src/App/App.test.tsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ describe('App', () => {
     expect(searchBar).toBeInTheDocument();
   })
 
-  xit('should show various pages when nav links are clicked', () => {
+  it('should show various pages when nav links are clicked', () => {
```
Comment (leighlars) [inline src/App/App.test.tsx:18]: nice! attaway to make it pass

File: src/Card/Card.scss [hunk 4]
```diff
@@ -0,0 +1,36 @@
+@import '../styles/_variables.scss';
+
+.card-container {
```
Comment (leighlars) [inline src/Card/Card.scss:3]: good deal! can't wait to start movin'

File: src/Card/Card.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Link } from 'react-router-dom'
+
+import './Card.scss'
+
+interface CardProps {
```
Comment (leighlars) [inline src/Card/Card.tsx:6]: will work on this this afternoon

**Review by leighlars — APPROVED** [review]
Great work, Josh! Thanks for getting the API proxy issue addressed.

File: src/Card/Card.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  name: string
+}
+
+const Card: React.FC<CardProps> = (props) => {
```
Comment (ErinUntermeyer) [inline src/Card/Card.tsx:10]: I like that we have seen multiple ways to do this so far! I will mimic this going forward but happy to explore the initial way you completed this as well as a refactor if you'd like.

File: src/helpers/airData.ts [hunk 7]
```diff
@@ -0,0 +1,104 @@
+export const airData = [
```
Comment (ErinUntermeyer) [inline src/helpers/airData.ts:1]: Thanks for making this so we don't have to continuously fetch the data while we style.

File: src/helpers/apiCalls.tsx [hunk 8]
```diff
@@ -1,28 +1,48 @@
...
+    const parseData = await response.json();
+    return parseData;
+  } catch (error) {
+    return error;
+  }
+};
+
+
+export const getAirQualityData = (lat: number, long: number) => {
+   return fetch("https://fe-cors-proxy.herokuapp.com", {
```
Comment (ErinUntermeyer) [inline src/helpers/apiCalls.tsx:35]: AWESOME JOB using the proxy!!!!!

**Pull request merged**